ui/src/app/utils/leo-runtime-initializer.spec.tsx

```diff
@@ -7,7 +7,7 @@ import {Runtime} from 'generated/fetch';
 import {RuntimeStatus} from 'generated/fetch';
 import {RuntimeApi} from 'generated/fetch/api';
 import SpyInstance = jest.SpyInstance;
-import expect = jest.Expect;
+import { expect } from '@jest/globals';
```

Comment:
nit: general convention in the codebase (and this file) is `{expect}` instead of `{ expect }`

Reply:
👍🏻